Update enterprise (#108)
solo-io/solo-docs: gloo/docs/installation/glooe.md

> * [*Ingress*](#ingress): Gloo will support configuration the Kubernetes Ingress resource, acting as a Kubernetes

Reviewer: ```suggestion
* [*Ingress*](#ingress): Gloo will support configurating the Kubernetes Ingress resource, acting as a Kubernetes
```

{{% notice info %}}

> Each Key is valid for **31 days**. You can request a new key if the current key that you have expired.

Reviewer: ```suggestion
Each Key is valid for **31 days**. You can request a new key if your current key expired.
```

> You will only require your License Key during the installation process. Once you install, a `secret` will be created to hold your unique key.

Reviewer: ```suggestion
The License Key is required only during the installation process. Once you install, a `secret` will be created to hold your unique key.
```
